docs/source/onedal/appendix/decision-tree.rst

```diff
@@ -0,0 +1,60 @@
+.. ******************************************************************************
+.. * Copyright 2021 Intel Corporation
+.. *
+.. * Licensed under the Apache License, Version 2.0 (the "License");
+.. * you may not use this file except in compliance with the License.
+.. * You may obtain a copy of the License at
+.. *
+.. *     http://www.apache.org/licenses/LICENSE-2.0
+.. *
+.. * Unless required by applicable law or agreed to in writing, software
+.. * distributed under the License is distributed on an "AS IS" BASIS,
+.. * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. * See the License for the specific language governing permissions and
+.. * limitations under the License.
+.. *******************************************************************************/
+
+.. _decision_tree:
```

Comment:
there is already a topic with this label in `docs\source\daal\algorithms\decision_tree\decision-tree.rst`, which results in an error during the build process

as a possible (temporary) solution:

```suggestion
.. _dt:
```

plus you'd need to make changes in text where you reference this topic